omnigibson/systems/micro_particle_system.py

```diff
@@ -1616,6 +1617,15 @@ def clothify_mesh_prim(cls, mesh_prim, remesh=True, particle_distance=None):
             mesh_prim.GetAttribute("normals").Set(lazy.pxr.Vt.Vec3fArray.FromNumpy(new_normals))
             mesh_prim.GetAttribute("primvars:st").Set(lazy.pxr.Vt.Vec2fArray.FromNumpy(new_texcoord))
 
+            # Load the new mesh into trimesh and apply the inverse of the world transform
```

Comment:
Right now this code does pymeshlab -> omni -> trimesh -> omni. Can we change that to pymeshlab -> trimesh -> omni? Essentially instead of the stuff in 1607-1612 we would first create the trimesh and then get the data from trimesh.